### ISSUE TITLE
Alert close button does not work

### ISSUE THREAD

**Issue by brunosotto** [RocketChat/Rocket.Chat#9387]
Translated:

When I try to send a file and it exceeds the limit in mb a red stripe is displayed at the top
File exceeds allowed size of 2 B. [error-file-too-large]
But clicking the close button does not close it.

Original
---
Quando tento enviar um arquivo e ele excede o limite em mb é exibido uma tarja vermelha no topo 
File exceeds allowed size of 2 B. [error-file-too-large]
Porém ao clicar no botão fechar ele não fecha.

**Comment by geekgonecrazy** [comment]
possible duplicate of: #7220

What version of Rocket.Chat are you using?

**Comment by hexcsl** [comment]
The same problem with version 0.60.3 and 0.60.4. I can not close the pop-up notification. Disappears only after the refresh page or restart desktop client (for Windows).

**Comment by localguru** [comment]
Same here on 0.61.0.
